app/uk/gov/hmrc/transitmovements/controllers/MovementsController.scala

```diff
@@ -135,6 +142,37 @@ class MovementsController @Inject() (
       case None    => updateMovementLargeMessage(movementId, triggerId)
     }
 
+  def updateMessage(movementId: MovementId, messageId: MessageId) =
+    Action.async(parse.json) {
+      implicit request =>
+        mapRequest(request.body)
+          .flatMap {
+            largeMessageMetadata => handleUpdateMessage(largeMessageMetadata, movementId, messageId)
```

Comment:
largeMessageMetadata, could be a small message so can we rename it to messageMetadata instead?